test/e2e/upgrade_baremetal_operator.go

```diff
@@ -17,6 +17,7 @@ type upgradeBMOInput struct {
 	SpecName          string
 }
 
+// upgradeIronic upgrade BMO image to the latest.
```

Comment:
A typo here, should likely be upgradeBMO

Reply:
copy paste 😄